advanced/firmware/src/bin/vec.rs

```diff
@@ -16,13 +17,16 @@ fn main() -> ! {
 
     // `heapless::Vec` exposes the same API surface as `std::Vec` but some of its methods return a
     // `Result` to indicate whether the operation failed due to the `heapless::Vec` being full
-    log::info!("start: {:?}", buffer);
+    defmt::info!("start: {:?}", defmt::Debug2Format(&buffer));
+    //                                 ^^^^^^^^^^^^^^^^^^^ this adapter is currently needed to log
+    //                                                     `heapless` data structures (like `Vec` here)
+    //                                                     with `defmt`
```

Comment:
```suggestion
    //                                 ^^^^^^^^^^^^^^^^^^^ this adapter is currently needed to log
    //                                                     `heapless` data structures (like `Vec` here)
    //                                                      with `defmt`
```

Reply:
nope the indentation is correct, the `at the 2nd line just makes it confusing